modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -6,10 +6,7 @@
 [id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
 = Creating a route through an Ingress object
 
-Some ecosystem components have an integration with Ingress resources but not with
-route resources. To cover this case, {product-title} automatically creates
-managed route objects when an Ingress object is created. These route objects are deleted
-when the corresponding Ingress objects are deleted.
+Some ecosystem components have an integration with Ingress resources but not with route resources. To cover this case, {product-title} automatically creates managed route objects when an Ingress object is created. These route objects are deleted when the corresponding Ingress objects are deleted.
```

Comment:
You could change "Ingress resources" to "`Ingress` resources", given that it relates to a resource. The same applies for "Ingress object", which could become "`Ingress` object".

The same might potentially apply to "route object". In addition to back ticks, you could also use uppercase "`Route`".